spk/ffmpeg/Makefile

```diff
@@ -1,8 +1,8 @@
 SPK_NAME = ffmpeg
 SPK_VERS = 4.3.2
-SPK_REV = 37
+SPK_REV = 38
 SPK_ICON = src/ffmpeg.png
-CHANGELOG = "1. Update to FFMPEG 4.3.2<br/>2. Update Intel libva to version 2.10 and media drivers 2021Q1<br/>3. Multiple depedency updates (libass, SVT-AV1, libaom, x264, libbluray, libvpx)"
+CHANGELOG = "Added zvbi library"
```

Comment:
Please add "1." as I will most probably make additional changes along the way.

Reply:
Ok